src/beman/execution/CMakeLists.txt

```diff
@@ -193,12 +201,20 @@ source_group("Header Files\\detail" FILES ${DETAIL_HEADER_FILES})
 
 set_target_properties(${TARGET_NAME} PROPERTIES VERIFY_INTERFACE_HEADER_SETS ON)
 
-target_compile_features(${TARGET_NAME} PUBLIC
-  "$<$<COMPILE_FEATURES:cxx_std_26>:cxx_std_26>"
-  "$<$<NOT:$<COMPILE_FEATURES:cxx_std_26>>:cxx_std_23>"
-)
+# Check for C++26 support at configuration time instead of using target_compile_features
+include(CheckCXXCompilerFlag)
+check_cxx_compiler_flag("-std=c++26" COMPILER_SUPPORTS_CXX26)
+check_cxx_compiler_flag("-std=c++23" COMPILER_SUPPORTS_CXX23)
+
+if(COMPILER_SUPPORTS_CXX26)
+  message(STATUS "Using C++26 for ${TARGET_NAME}")
+elseif(COMPILER_SUPPORTS_CXX23)
+  message(STATUS "Using C++23 for ${TARGET_NAME}")
+else()
+  message(FATAL_ERROR "The compiler ${CMAKE_CXX_COMPILER} does not support C++23 or later. Please use a different compiler.")
+endif()
```

Comment:
Delete this (but deleting what was there was a good change; we just didn't need to add anything)